src/fiatconnect/index.test.ts

```diff
@@ -134,4 +161,57 @@ describe('FiatConnect helpers', () => {
       ).rejects.toThrowError('Not implemented')
     })
   })
+  describe('loginWithFiatConnectProvider', () => {
+    const wallet = (mocked(KeychainWallet, true) as unknown) as jest.Mocked<KeychainWallet>
```

Comment:
This looks very hacky, is there a better way to do this? Maybe just create a `KeychainWallet` / `FiatConnectClient` instances with dummy args since you've already mocked out the imports?

Reply:
Yeah, I meant to add a comment here; this is pending a fix at the moment. Will update with something more correct.